### ISSUE TITLE
Add FAQ/Wiki/Recipes 

### ISSUE THREAD

**Issue by jrodbx** [cashapp/paparazzi#583]
Ideas:
- how to target a specific SDK level in Paparazzi
- move git lfs section out of main page to here

**Comment by yschimke** [comment]
It might be worth making the error message when lfs isn't set up more obvious in code.

Instead of the NPE, detect automatically or catch and check if the file is an lfs marker file.

**Comment by jrodbx** [comment]
> It might be worth making the error message when lfs isn't set up more obvious in code.

Using Git LFS is an implementation detail, so prob shouldn't be in code?  For example, I've heard some users redirecting snapshots to an NFS instead.

**Comment by yschimke** [comment]
Yep, but might improve the errors when it happens. Maybe generic, turn the NPE into a clearer error. "File is corrupt"?
